users/views.py

```diff
@@ -1,9 +1,27 @@
 from rest_framework import generics
-from rest_framework.permissions import AllowAny
+from rest_framework.permissions import AllowAny, IsAuthenticated
 
-from users.serializers import SignupSerializer
+from users.models import User
+from users.serializers import SignupSerializer, UpdateSerializer, UserPrivateSerializer
 
 
 class SignupView(generics.CreateAPIView):
     permission_classes = (AllowAny,)
     serializer_class = SignupSerializer
+
+
+class GetView(generics.RetrieveAPIView):
+    queryset = User.objects.all()
+    permission_classes = (IsAuthenticated,)
```

Comment:
We need object permissions so that user can update only their own profile

Reply:
Done